data-acquisition/data-synthesizer/src/template_manager/template_manager.py

```diff
@@ -0,0 +1,146 @@
+"""
+Module to manage the addition, deletion and modification of foreground
+and background images inside the img_templates folder. The configurations
+will be stored in the files in config folder.
+"""
+
+import os
+import json
+
+
+class TemplateManager:
+    def __init__(self, config_path):
+        self.config_path = config_path
+
+    def add_foreground_category(self, category):
+        """
+        Adding new foreground category.
+        """
+        config_path = self.config_path + "foregrounds.json"
+        with open(config_path, "r") as conf_file:
+            config = json.load(conf_file)
+        if category not in config:
+            config[category] = {}
+        else:
+            raise Exception((
+                "Category {category} present in foreground configuration."
+            ).format(category=category))
+
+        with open(config_path, "w") as conf_file:
+            json.dump(config, conf_file, indent=4)
+        print("New foreground category {} added.".format(category))
+
+    def del_foreground_category(self, category):
+        """
+        Removing foreground category.
+        """
+        config_path = self.config_path + "foregrounds.json"
+        with open(config_path, "r") as conf_file:
+            config = json.load(conf_file)
+        if category in config:
```

Comment:
Same as [this comment](https://github.com/havanagrawal/clomask/pull/22/files#r246612494)